packages/@vivliostyle/theme-base/css/partial/page.css

```
@@ -0,0 +1,534 @@
+:root {
+  --vs-page--size: auto;
+  --vs-page--width: auto;
+  --vs-page--height: auto;
+  --vs-page--marks: none;
+  --vs-page--bleed: 0;
+  --vs-page--color-bg: var(--vs-color-bg);
+  --vs-page--margin-top: 22mm;
+  --vs-page--margin-bottom: 22mm;
+  --vs-page--margin-inner: 18mm;
+  --vs-page--margin-outer: 18mm;
+  --vs-page--mbox-color-bg: transparent;
+  --vs-page--mbox-color-body: inherit;
+  --vs-page--mbox-content-top-left-corner: '';
+  --vs-page--mbox-content-top-left: '';
+  --vs-page--mbox-content-top-center: '';
+  --vs-page--mbox-content-top-right: '';
+  --vs-page--mbox-content-top-right-corner: '';
+  --vs-page--mbox-content-left-top: '';
+  --vs-page--mbox-content-left-middle: '';
+  --vs-page--mbox-content-left-bottom: '';
+  --vs-page--mbox-content-right-top: '';
+  --vs-page--mbox-content-right-middle: '';
+  --vs-page--mbox-content-right-bottom: '';
+  --vs-page--mbox-content-bottom-left-corner: '';
+  --vs-page--mbox-content-bottom-left: '';
+  --vs-page--mbox-content-bottom-center: '';
+  --vs-page--mbox-content-bottom-right: '';
+  --vs-page--mbox-content-bottom-right-corner: '';
```

```suggestion
  --vs-page--mbox-content-top-left-corner: none;
  --vs-page--mbox-content-top-left: none;
  --vs-page--mbox-content-top-center: none;
  --vs-page--mbox-content-top-right: none;
  --vs-page--mbox-content-top-right-corner: none;
  --vs-page--mbox-content-left-top: none;
  --vs-page--mbox-content-left-middle: none;
  --vs-page--mbox-content-left-bottom: none;
  --vs-page--mbox-content-right-top: none;
  --vs-page--mbox-content-right-middle: none;
  --vs-page--mbox-content-right-bottom: none;
  --vs-page--mbox-content-bottom-left-corner: none;
  --vs-page--mbox-content-bottom-left: none;
  --vs-page--mbox-content-bottom-center: none;
  --vs-page--mbox-content-bottom-right: none;
  --vs-page--mbox-content-bottom-right-corner: none;
```

margin box の content が none なら margin box が生成されないが、空文字列 "" の場合は margin box を生成する無駄な処理が生じます。また、ページヘッダーに左寄せで１つだけ柱を置くような場合、top-center の margin box が生成されると top-left の margin box の最大幅が制限されてしまう問題があります。